modules/DesktopStreaming/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>gephi-plugin-parent</artifactId>
+        <groupId>org.gephi</groupId>
+        <version>0.9.0</version>
+    </parent>
+
+    <groupId>org.gephi.desktop</groupId>
```

Comment:
You should pick a groupId different from "org.gephi" as we want to reserve this to official Gephi plugins. I would also suggest to revise your naming strategy so your groupId is the same for all of the plugin's modules, and then vary the artifactId, for instance 'streaming-plugin', 'streaming-api', 'desktop-streaming' etc.